core/src/tile/labels/label.cpp

```diff
@@ -0,0 +1,41 @@
+#include "label.h"
+
+Label::Label(LabelTransform _transform, std::string _text, std::shared_ptr<TextBuffer> _buffer) :
+    m_transform(_transform),
+    m_text(_text),
+    m_buffer(_buffer) {
+
+    m_id = m_buffer->genTextID();
+}
+
+Label::~Label() {}
+
+void Label::rasterize() {
+    m_buffer->rasterize(m_text, m_id);
+}
+
+void Label::updateTransform(const LabelTransform& _transform, const glm::mat4& _mvp, const glm::vec2& _screenSize) {
+    m_transform = _transform;
```

Comment:
No use of having a LabelTransform argument here.

If ever transforms of a label needs any changing, another method could be used to set that ("setLabelTransform").